packages/amplify-nodejs-function-template-provider/resources/lambda/crud/package.json.ejs

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "<%= props.functionName.toLowerCase() %>",
+"name": "<%= props.functionName.toLowerCase() %>",
```

Comment:
The changes to this file should be reverted because we might have snapshot tests that depend on the output, and these spacing changes get reflected in what we generate IIRC.